uberfire-extensions/uberfire-layout-editor/uberfire-layout-editor-backend/src/main/java/org/uberfire/ext/layout/editor/impl/PerspectiveServicesImpl.java

```diff
@@ -137,4 +150,21 @@ private void setLayoutTemplateName(Path path, String newName, String comment) {
         LayoutEditorModel pluginCopy = new LayoutEditorModel(newName, PluginType.PERSPECTIVE_LAYOUT, path, layoutModel);
         pluginServices.saveLayout(pluginCopy, comment);
     }
+
+    @Override
+    public Path save(final Path path,
+                     final LayoutTemplate content,
+                     final DefaultMetadata metadata,
+                     final String comment) {
+        return saveLayoutTemplate(path, content, comment);
```

Comment:
Sorry for question, can't be this done as?:
```java
saveAndRenameService.save(path, content, metadata, comment);
```

Reply:
(No problem) ;-) ..and no, it can't be done. The `SaveAndRenameServiceImpl` is initialised with:
```java
saveAndRenameService.init(this); // this is an instance of PerspectiveServices
```
This `init` method sets the `SaveAndRenameService.updateService` 
and the:
```java
saveAndRenameService.save(_path, content, _metadata, comment);
```
would call
```java
updateService.save(path, content, metadata, comment)
```
Take a deeper look at `SaveAndRenameServiceImpl.java`, maybe it will clarify things even more.